### DISCUSSION TITLE
Multiple schemas with hasura local console.

### DISCUSSION THREAD

**Discussion by SirClutch6** [hasura/graphql-engine#8739] [Question]
Using nhost and Hasura, has anyone been able to create schemas apart from "public" and get them to work properly in a local Hasura console? It seems that adding any additional schemas causes the GraphQL Engine metadata to be inconsistent with the database.

![Screenshot from 2022-08-03 11-35-06](https://user-images.githubusercontent.com/61561983/182673854-926c202f-a385-4613-82d5-7fe3ab79a678.png)
![Screenshot from 2022-08-03 11-35-36](https://user-images.githubusercontent.com/61561983/182673856-87e95a88-ebd5-4ff7-acf2-faa236a43011.png)
.

**Comment by adas98012** [comment]
Please log a Github issue, with detailed steps on how to reproduce the problem you are encountering. That will help our engineering team to troubleshoot the issue and find a solution.
